.github/workflows/provisioning-tests.yaml

```diff
@@ -0,0 +1,58 @@
+name: Provisioning tests
+
+on:
+  push: 
+    branches:
+      - 'dev-v*'
+      - 'release-v*'
+  pull_request:
+    branches:
+      - 'dev-v*'
+      - 'release-v*'
+
+jobs:
+  provisioning-test:
+    permissions:
+      contents: read
+    runs-on: ubuntu-latest
+    container: 
+      image: rancher/dapper:v0.6.0
+      options: --privileged
+    timeout-minutes: 60
+    strategy:
+      matrix:
+        dist: [rke2, k3s]
+        k8s-minor: [25, 26, 27, 28]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Provisioning test
+        run: |
+          dapper provisioning-tests
+        env:
+          V2PROV_TEST_DIST: ${{ matrix.dist }}
+          V2PROV_TEST_RUN_REGEX: "^Test_Provisioning_.*$"
+          KDM_TEST_K8S_MINOR: ${{ matrix.k8s-minor }}
+
+  provisioning-operations-test:
+    permissions:
+      contents: read
+    runs-on: ubuntu-latest
+    container: 
+      image: rancher/dapper:v0.6.0
+      options: --privileged
+    timeout-minutes: 60
+    strategy:
+      matrix:
+        dist: [rke2, k3s]
+        k8s-minor: [25, 26, 27, 28]
```

Comment:
```suggestion
        k8s-minor: [25, 26, 27, 28, 29]
```

Reply:
added both 29 and 30 and removed 25 and 26 as per supported version list